src/ctelnet.h

```diff
@@ -258,6 +273,37 @@ public slots:
     int decompressBuffer(char*& in_buffer, int& length, char* out_buffer);
     void reset();
 
+    QString encodeNewEnvironData(const QString&);
+    QString escapeNewEnvironData(const QString&);
+    QString getNewEnvironValueUser();
+    QString getNewEnvironValueSystemType();
+    QString getNewEnvironUserValueCharset();
+    QString getNewEnvironUserValueClientName();
+    QString getNewEnvironUserValueClientVersion();
+    QString getNewEnvironUserValueTerminalType();
+    QString getNewEnvironUserValueMTTS();
+    QString getNewEnvironUserValueIPAddress();
+    QString getNewEnvironUserValueANSI();
+    QString getNewEnvironUserValueVT100();
+    QString getNewEnvironUserValue256Colors();
+    QString getNewEnvironUserValueMouseTracking();
+    QString getNewEnvironUserValueUTF8();
+    QString getNewEnvironUserValueOSCColorPalette();
+    QString getNewEnvironUserValueScreenReader();
+    QString getNewEnvironUserValueProxy();
+    QString getNewEnvironUserValueTruecolor();
+    QString getNewEnvironUserValueTLS();
+    QString getNewEnvironUserValueLanguage();
+    QString getNewEnvironUserValueFont();
+    QString getNewEnvironUserValueFontSize();
+    QString getNewEnvironUserValueWordWrap();
+    std::string appendAllNewEnvironValues(std::string&, const bool, const QMap<QString, QPair<bool, QString>>&);
+    std::string appendNewEnvironValue(std::string&, const QString&, const bool, const QMap<QString, QPair<bool, QString>>&);
```

Comment:
To avoid these being impropriately used like `output += appendAllNewEnvironValues(output, ...)` which does a "double append", make them void:

```suggestion
    void appendAllNewEnvironValues(std::string&, const bool, const QMap<QString, QPair<bool, QString>>&);
    void appendNewEnvironValue(std::string&, const QString&, const bool, const QMap<QString, QPair<bool, QString>>&);
```